node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const desiredFile = process.argv[2];
+
+console.log(desiredFile);
+
+fs.readFile(desiredFile, (err, data) => {
+  if (err) throw (err);
```

Comment:
(Optional) Syntactically, the () around err in `throw (err)` aren't necessary. Keep them if it looks better to you. Personally, I don't use them in `throw`.